tests/Oryx.RuntimeImage.Tests/Php-fpm/PhpFpmImageTest.cs

```diff
@@ -178,7 +178,6 @@ public void Redis_IsInstalled(string imageTag)
         [InlineData("8.2-fpm")]
         [InlineData("8.1-fpm")]
         [InlineData("8.0-fpm")]
-        [InlineData("7.4-fpm")]
```

Comment:
Any reason why this test was removed?

Reply:
in this test it actually checks if the `pdo_sqlsrv` is installed on the image. But `pdo_sqlsrv` requires php version above 8.0. hence, I removed the installation of `pdo_sqlsrv` on the image. Therefore, testing `pdo_sqlsrv` in php version below 8.0 is pointless. 
I believe App service team does not use php version lower than 8.0 anymore. For that reason, we are now considering not to create images with php lower than 8.0. Dan will probably create a work item for that.